zulipterminal/ui_tools/views.py

```diff
@@ -651,10 +652,19 @@ def users_view(self, users: Any=None) -> Any:
         if users is None:
             users = self.view.users.copy()
             reset_default_view_users = True
+
+        state_icon = {
+            'active': STATUS_ACTIVE,
+            'idle': STATUS_IDLE,
+            'offline': STATUS_OFFLINE,
+            'inactive': STATUS_INACTIVE,
+        }
+
         users_btn_list = list()
         for user in users:
+            status = user['status']
```

Comment:
You seem to have done a small refactor here, but I guess it is small enough to integrate with the other change?

Reply:
This is indeed small, and within a function it should be safe enough :+1: